packages/worker/src/api/routes/global/email.js

```diff
@@ -12,7 +12,13 @@ function buildEmailSendValidation() {
   return joiValidator.body(Joi.object({
     email: Joi.string().email({
       multiple: true,
-      }),
+    }),
+    cc: Joi.string().email({
+      multiple: true,
+    }),
+    bcc: Joi.string().email({
+      multiple: true,
+    }),
```

Comment:
should these have `.optional()` appended?

Reply:
Nice catch! I tried optional but it doesn't change the behaviour. Changed to `allow("", null)` instead to match the other 'optional' fields.

Reply:
just to confirm, can we send a request to this endpoint with those fields missing completely?
that's odd on optional, it usually works. Did you try it like this?
```ts
Joi.string().email().optional()
```

Reply:
Yeah that's how I tried it, but it was already optional without it. The allow `null` means that null values can be sent through.

The following scenarios tested to work, the request went through:
 - Blank fields
 - JavaScript returning `undefined`
 - JavaScript returning `null`

Reply:
sorry just to confirm again on Blank fields, do you mean with the value `""` or with the field omitted completely?

Reply:
both!